low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -116,6 +116,7 @@ NODEHWSENSOR:
    threaded: true
    polling_interval: 30
    ipmi_client: ipmitool
+   default_server_frus: ['disk', 'psu', 'fan']
```

Comment:
There are no default FRUs, we are giving option to extend that FRU list by declaring explicitly, so proper name is needed
'declare_server_fru' or 'extend_server_fru_list'

ALso, avoid fan in list, as per my previous comment

Reply:
okay will rename the list name.
one question, if 'fan' is not declared in the list,
then for fan sensor resource (node:hw:fan) , the fru flag will be false
"fru": false
because server fru list is not giving fan as a fru, neither it will find in server_declare_list.

Reply:
As discussed, removing 'fan' fru from server_declare_list